### ISSUE TITLE
Add compiled bindings support

### ISSUE THREAD

**Issue by taori** [Microsoft/xaml-standard#102]
Talking about this feature

https://docs.microsoft.com/de-de/windows/uwp/xaml-platform/x-bind-markup-extension

Please make it available in the Standard.

**Comment by mossyblog** [comment]
Agreed. Adobe FLEX uses attributeName="{ Foo.x + 1}" and its essentially asserted that Foo.X + 1 is parsed as inline directive(s).

I'm not sure what value attribute="{ x:Bind Foo.x +1}" provides given the nature of  the syntax "{ }" would imply "evaluate" behaviours. x:Bind would infer that the property would first have to be derived from bindable property class implementations which has the ye olde "NotifyPropertyChanged()" binding path implied... where as it really does depend on the platforms attitude towards how it invokes binding(s) ...whether it be "wait for a property to change" or "every frame look for property change"

**Comment by birbilis** [comment]
maybe better to post the English url
https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension
